src/commands/doctor.ts

```diff
@@ -95,9 +95,24 @@ module.exports = {
       expoVersion = "-"
       expoWorkflow = "not installed"
     }
-
     const expoInfo = [column1("expo"), column2(expoVersion), column3(expoWorkflow)]
 
+    let depExpoCLIInfo = null
```

Comment:
```suggestion
    let depExpoCliInfo = null
```

nit: just for consistency across the project, some casing preferences